source/_includes/common-tasks/backups.md

```diff
@@ -41,7 +41,8 @@ The automatic backup process creates a backup on a predefined schedule and also
    - It is recommended to back up **Daily**, but you can also choose to back up on specific days.
    - Define the time:
      - **System optimal** sets a time in a predefined time window as shown in the UI.
-     - **Custom**: Pick the time when you want the backup to start.
+     - **Custom** lets you pick the time when you want the backup to start.
+     - Make sure you pick a time when all your backup locations are up and running and available. Otherwise, the backup will fail for the location that is not available.
```

Comment:
```suggestion
     - Make sure you pick a time when all your backup locations are up and running and available. Otherwise, the backup will fail for locations which are not available.
```